model/src/main/java/org/cbioportal/model/Gene.java

```diff
@@ -49,4 +51,26 @@ public Integer getLength() {
     public void setLength(Integer length) {
         this.length = length;
     }
+    
+    public String getChromosome() {
```

Comment:
Ok, I'm putting in an issue into the issue tracker for discussion. for now, these will be on "X"